lmdeploy/pytorch/models/internvl.py

```diff
@@ -138,9 +196,7 @@ def forward(self, hidden_states):
         q, k, v = self.qkv.split_qkv(qkv_states)
 
         if self.qk_normalization:
-            q_shape = q.shape
-            q = self.q_norm(q.flatten(-2, -1)).view(q_shape)
-            k = self.k_norm(k.flatten(-2, -1)).view(q_shape)
+            q, k = self.qkv_norm(q, k)
```

Comment:
could fix the hf version as well.

https://github.com/InternLM/lmdeploy/blob/d87a847be3ef8461a66205a068253542b395aefc/lmdeploy/pytorch/models/internvl3_hf.py#L193